src/routes/Home.jsx

```diff
@@ -0,0 +1,17 @@
+import Header from '@/components/Header';
+import TodosLogic from '@/components/TodosLogic';
+
+const Home = () => {
+  return (
+    // <div className="wrapper"> remove the div
```

Comment:
hello @mrcbq may i suggest removing commented code, because it goes against the best practices.
```suggestion
    // <div className="wrapper"> remove the div
```

Reply:
It's done.